webapp/packages/plugin-administration/src/ConfigurationWizard/ServerConfiguration/ServerConfigurationFormPart.ts

```diff
@@ -0,0 +1,144 @@
+ */
+import { AdministrationScreenService } from '@cloudbeaver/core-administration';
+import { ADMIN_USERNAME_MIN_LENGTH, AUTH_PROVIDER_LOCAL_ID, AuthProvidersResource, PasswordPolicyService } from '@cloudbeaver/core-authentication';
+import { DEFAULT_NAVIGATOR_VIEW_SETTINGS } from '@cloudbeaver/core-connections';
+import { ExecutorInterrupter, type IExecutionContextProvider } from '@cloudbeaver/core-executor';
+import { CachedMapAllKey } from '@cloudbeaver/core-resource';
+import { DefaultNavigatorSettingsResource, PasswordPolicyResource, ProductInfoResource, ServerConfigResource } from '@cloudbeaver/core-root';
+import { FormPart, IFormState } from '@cloudbeaver/core-ui';
+import { isObjectsEqual } from '@cloudbeaver/core-utils';
+
+import { MIN_SESSION_EXPIRE_TIME } from './Form/MIN_SESSION_EXPIRE_TIME';
+import { IServerConfigurationFormPartState } from './IServerConfigurationFormPartState';
+
+function DEFAULT_STATE_GETTER(): IServerConfigurationFormPartState {
+  return {
+    serverConfig: {
+      adminCredentialsSaveEnabled: false,
+      anonymousAccessEnabled: false,
+      authenticationEnabled: false,
+      customConnectionsEnabled: false,
+      disabledDrivers: [],
+      enabledAuthProviders: [],
+      enabledFeatures: [],
+      publicCredentialsSaveEnabled: false,
+      resourceManagerEnabled: false,
+      serverName: '',
+      serverURL: '',
+      sessionExpireTime: MIN_SESSION_EXPIRE_TIME * 1000 * 60,
+    },
+    navigatorConfig: { ...DEFAULT_NAVIGATOR_VIEW_SETTINGS },
+  };
+}
+
+export class ServerConfigurationFormPart extends FormPart<IServerConfigurationFormPartState> {
+  constructor(
+    formState: IFormState<null>,
+    private readonly administrationScreenService: AdministrationScreenService,
+    private readonly serverConfigResource: ServerConfigResource,
+    private readonly productInfoResource: ProductInfoResource,
+    private readonly defaultNavigatorSettingsResource: DefaultNavigatorSettingsResource,
+    private readonly authProvidersResource: AuthProvidersResource,
+    private readonly passwordPolicyResource: PasswordPolicyResource,
+    private readonly passwordPolicyService: PasswordPolicyService,
+  ) {
+    super(formState, DEFAULT_STATE_GETTER());
+  }
+
+  isOutdated(): boolean {
+    return super.isOutdated() || this.serverConfigResource.isOutdated() || this.defaultNavigatorSettingsResource.isOutdated();
+  }
+
+  isLoaded(): boolean {
+    return super.isLoaded() && this.serverConfigResource.isLoaded() && this.defaultNavigatorSettingsResource.isLoaded();
+  }
+
+  protected async validate(
+    data: IFormState<IServerConfigurationFormPartState>,
+    contexts: IExecutionContextProvider<IFormState<IServerConfigurationFormPartState>>,
+  ) {
+    if (this.administrationScreenService.isConfigurationMode) {
+      await this.authProvidersResource.load(CachedMapAllKey);
+
+      if (this.authProvidersResource.has(AUTH_PROVIDER_LOCAL_ID)) {
+        await this.passwordPolicyResource.load();
```

Comment:
You preloaded the resource, but it is not used in this current form part. Seems like it can be removed from here completely

Reply:
<img width="678" alt="image" src="https://github.com/user-attachments/assets/1b7cfcce-91b6-4691-9288-9c8042222868">

it should be preloaded so validatePassword is working correctly